src/libsync/syncfileitem.h

```diff
@@ -252,7 +252,7 @@ class OWNCLOUDSYNC_EXPORT SyncFileItem
     // usually this value is 1, but for removes on dirs, it might be much higher.
 
     // Variables used by the propagator
-    csync_instructions_e _instruction;
+    SyncInsturctions _instruction;
```

Comment:
As far as i know, the item can only have only one instruction at the time.
This should be a `SyncInsturction`, not the QFlags.

Reply:
Well I can't come up with many use cases besides for tests like
`ENFORCE(instruction & (NEW | REMOVE))`
But yes maybe we should not encourage that.